insights/specs/sos_archive.py

```diff
@@ -274,6 +274,7 @@ class SosSpecs(Specs):
     swift_log = first_file(["/var/log/containers/swift/swift.log", "/var/log/swift/swift.log"])
     sys_kernel_sched_features = simple_file("/sys/kernel/debug/sched_features")
     sysconfig_chronyd = simple_file("/etc/sysconfig/chronyd")
+    sysconfig_grub = simple_file("/etc/sysconfig/grub")
```

Comment:
For `simple_file`, if the path is the same as it is on the real system, the definition in `default.py` works for all the archives (sos, insights archive) since the archives keep their original path. This line can be removed.